docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -299,19 +418,16 @@ To decode a JPEG, PNG, GIF, or BMP image:
 
    > **Note**
    >
-   > - Because of decoder limitations, color space setting and JPEG downscaling are only supported for JPEG images.
-   > - The default color space is `IMAGE_UTIL_COLORSPACE_RGBA8888`. PNG, GIF and BMP images are decoded to `IMAGE_UTIL_COLORSPACE_RGBA8888`.
+   > Due to the encoder limitations, the color space setting is only supported for encoding the JPEG images. The default color space is `IMAGE_UTIL_COLORSPACE_RGBA8888`. PNG, GIF, and BMP images are encoded with `IMAGE_UTIL_COLORSPACE_RGBA8888`.
```

Comment:
Just for check. Is this note really talking about encoding limitation while the note itself is located in decoding section?
Moreover the same note is located in encoding section, too.

Reply:
It's my mistake, it should be decoder limitation. Iit has same limitation for decoding and encoding the image.
I will replace the sentence.